src/v/cluster/controller_backend.cc

```diff
@@ -1229,7 +1229,8 @@ controller_backend::process_partition_reconfiguration(
      * created with cancel/abort type of deltas.
      */
     vassert(
-      type == topic_table_delta::op_type::update,
+      type == topic_table_delta::op_type::update
+        || type == topic_table_delta::op_type::force_update,
```

Comment:
nit: should that land in a separate commit ?

Reply:
done.